src/db-service.ts

```diff
@@ -117,7 +117,7 @@ export class RecycledItemService {
     return transactionHandler
       .query<Item>(
```

Comment:
Nitpick: for consistency of types between SQL and TS, since you now project on `id`
```suggestion
      .query<Pick<Item, "id">>(
```